### PR TITLE
casa basement docs

### PR DESCRIPTION
**Pull request by bwright85428**
Adds basement documentation to Casa manual

### CODE REVIEW

**Review by benagricola — APPROVED** [review]
This looks good to me purely from the code, but I haven't contributed to the Casa manual so maybe @EdwardChamberlain will be better placed to point out any issues.

@bwright85428 it would be helpful if you can push these changes to a branch on the main repository, which will trigger a deployment so these changes can be previewed online (say for example you push from your `main` branch to the `casa-basement` branch on the main repository, the deployment would appear at https://millenniummachines.github.io/docs-casa-basement

**Comment by bwright85428** [comment]
Done. See [https://millenniummachines.github.io/docs-casa-basement/](https://millenniummachines.github.io/docs-casa-basement/) for the branch deployment

File: docs/casa/manual/chapters/50_basement.md [hunk 1]
```diff
@@ -4,47 +4,143 @@
...
+
 ## Feet
 
 !!! info annotate "Components Required"
     ```
-        ??
+        2x Leg A
+        2x Leg B
+        4x M3x3mm Heat Insert
+        4x Rubber Foot
```
Comment (benagricola) [inline docs/casa/manual/chapters/50_basement.md]:
```suggestion
        4 x M3x3mm Heat-set Insert
        4 x Rubber Foot
        2 x Leg A
        2 x Leg B
```

File: docs/casa/manual/chapters/50_basement.md [hunk 2]
```diff
@@ -4,47 +4,143 @@
...
 ---
 
+## Skirts
+
+E-Skirt
 !!! info annotate "Components Required"
     ```
-        ??
+        2x Skirt E
+        4x M3x3mm Heat Insert
```
Comment (benagricola) [inline docs/casa/manual/chapters/50_basement.md]:
```suggestion
        4 x M3x3mm Heat-set Insert
        2 x Skirt E
```

File: docs/casa/manual/chapters/50_basement.md [hunk 3]
```diff
@@ -4,47 +4,143 @@
...
 ---
 
+## Chip Canal
+
+Chip Canal B
 !!! info annotate "Components Required"
     ```
-        ??
+        Chip Canal B
+        2x M3x3mm Heat Insert
```
Comment (benagricola) [inline docs/casa/manual/chapters/50_basement.md]:
```suggestion
        2 x M3x3mm Heat-set Insert
        1 x Chip Canal B
```

File: docs/casa/manual/chapters/50_basement.md [hunk 4]
```diff
@@ -4,47 +4,143 @@
...
+![](../img/50_CC_1.png)
 
 ---
 
-## Skirts
+Chip Canal C
+!!! info annotate "Components Required"
+    ```
+        Chip Canal C
+        4x M3x3mm Heat Insert
```
Comment (benagricola) [inline docs/casa/manual/chapters/50_basement.md]:
```suggestion
        4 x M3x3mm Heat-set Insert
        1 x Chip Canal C
```

File: docs/casa/manual/chapters/50_basement.md [hunk 5]
```diff
@@ -4,47 +4,143 @@
...
 
-## Chip Canal
+Chip Canal Assembly
+!!! info annotate "Components Required"
+    ```
+        Chip Canal A
+        Chip Canal B
+        Chip Canal C
+        Chip Canal D
+        10x M3x10mm-SHCS
```
Comment (benagricola) [inline docs/casa/manual/chapters/50_basement.md]:
```suggestion
       10 x M3x10mm SHCS
        1 x Chip Canal A
        1 x Chip Canal B
        1 x Chip Canal C
        1 x Chip Canal D
```

File: docs/casa/manual/chapters/50_basement.md [hunk 6]
```diff
@@ -4,47 +4,143 @@
...
+        Skirt-B
+        Skirt-C
+        Skirt-D
+        Skirt-E
+        Skirt-F
+        Skirt-G
+        Skirt-H
+        Skirt-I
+        44x M3x12mm-SHCS
+        2x M5x10mm-BHCS
```
Comment (benagricola) [inline docs/casa/manual/chapters/50_basement.md]:
```suggestion
       44 x M3x12mm SHCS
        4 x Skirt K
        4 x Skirt J
        4 x Skirt L
        2 x M5x10mm BHCS
        1 x Leg A
        1 x Leg B
        1 x Skirt A
        1 x Skirt B
        1 x Skirt C
        1 x Skirt D
        1 x Skirt E
        1 x Skirt F
        1 x Skirt G
        1 x Skirt H
        1 x Skirt I
```

Ordered by quantity

File: docs/casa/manual/chapters/50_basement.md [hunk 7]
```diff
@@ -4,47 +4,143 @@
...
+        Skirt-A
+        Skirt-B
+        Skirt-C
+        Skirt-D
+        Skirt-E
+        Skirt-G
+        Skirt-H
+        Skirt-I
+        18x M3x12mm-SHCS
+        2x M5x10mm-BHCS
```
Comment (benagricola) [inline docs/casa/manual/chapters/50_basement.md]:
```suggestion
        18 x M3x12mm SHCS
         2 x M5x10mm BHCS
         1 x Leg A
         1 x Leg B
         1 x Skirt A
         1 x Skirt B
         1 x Skirt C
         1 x Skirt D
         1 x Skirt E
         1 x Skirt G
         1 x Skirt H
         1 x Skirt I
```

**Comment by benagricola** [comment]
> Done. See https://millenniummachines.github.io/docs-casa-basement/ for the branch deployment

Thanks :+1: 

On second read-through, added a bunch of suggestions that align a little more consistently with the rest of the docs - in short, consistent names and quantities for fixtures and printed parts.

**Comment by benagricola** [comment]
@bwright85428 is there anything outstanding here from your site stopping this from being merged?

**Comment by bwright85428** [comment]
@benagricola There are not any pending changes on my end. The casa-basement branch has the latest changes suggested by you and Ed. I don't think the PR is up to date

**Comment by benagricola** [comment]
Superceded by #81

**Pull request closed without merging**